app/controllers/cells_controller.rb

```diff
@@ -18,23 +18,24 @@ def create
         @cell = Cell.new(system: system)
         @cell.split
     end
-    if @cell.save
-      system.meta_points -= params["new cells"].to_i
-      system.save
-      flash[:notice] = "created #{params["new cells"].to_i} new cells"
-      redirect_to system_path(system)
+      if @cell.save
+        system.meta_points -= params["new cells"].to_i
+        system.save
+        flash[:notice] = "created #{params["new cells"].to_i} new cells"
+        redirect_to system_path(system)
+      else
+        render :new
+      end
     else
-      render :new
-    end
-  else
-    @cell = Cell.new(system_id: params[:system_id])
-    @cell.split
-    if @cell.save
-      flash[:notice] = 'cell created'
-      redirect_to system_path(system)
-    else
-      flash[:notice] = 'cell not created.'
-      render :new
+      @cell = Cell.new(system_id: params[:system_id])
+      @cell.split
+      if @cell.save
+        flash[:notice] = 'cell created'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.